sponsors.html

```diff
@@ -0,0 +1,300 @@
+    You can help make this an unforgettable event through financial or hardware donations, and by sending your
+    representatives to participate as judges or mentors.
+  </p>
+</div>
+
+<div style="padding-bottom: max(3em, 3vh);">
+  <h1 style="margin-bottom:1em">Sponsorship Tiers</h1>
+  <table class="sponsorship-table">
+    <!-- <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr> -->
+    <tr>
+      <th></th>
+      <th>Basic ($500)</th>
+      <th>Silver ($1000) </th>
+      <th>Gold ($2000)</th>
+      <th>Diamond ($3000+)</th>
+    </tr>
+
+    <!-- TODO: This should display a white border. Not showing up. Fix this issue. -->
+    <tr style="border-bottom:5px solid white">
+      <td colspan="100%"></td>
+    </tr>
+    <tr>
+      <th>Featured Logos</th>
+      <td>✔</td>
+      <td>✔</td>
+      <td>✔</td>
+      <td>✔</td>
+    </tr>
+    <tr>
+      <th>Presentation and Tabling</th>
+      <td>✔</td>
+      <td>✔</td>
+      <td>✔</td>
+      <td>✔</td>
+    </tr>
+    <tr>
+      <th>Resume Database</th>
+      <td></td>
+      <td>✔</td>
+      <td>✔</td>
+      <td>✔</td>
+    </tr>
+    <tr>
+      <th>DIY Tech Workshops</th>
+      <td></td>
+      <td></td>
+      <td>✔</td>
+      <td>✔</td>
+    </tr>
+    <tr>
+      <th>DIY Career Info Session</th>
+      <td></td>
+      <td></td>
+      <td>✔</td>
+      <td>✔</td>
+    </tr>
+    <tr>
+      <th>Company Challenges</th>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>✔</td>
+    </tr>
+  </table>
+</div>
+
+<div id="benefits-glossary">
+  <!-- TODO: Add padding above glossary -->
+  <h1 style="margin-bottom:1em">Benefits Glossary</h1>
+  <table class="benefits-table">
+    <tr>
+      <th>Benefit</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <th>Featured Logos</th>
+      <td>Your company logo will show up in our event publicity and marketing media, such as emails, flyers, and T-shirts</td>
+    </tr>
+    <tr>
+      <th>Presentation and Tabling</th>
+      <td>Your company can present (~5 minutes) during the opening ceremony and table for the whole hackathon</td>
+    </tr>
+    <tr>
+      <th>Resume Database</th>
+      <td>Your company will have access to the resumes of all participants</td>
+    </tr>
+    <tr>
+      <th>DIY Tech Workshops</th>
+      <td>Your company can host technical workshops for hackers and introduce them to any kind of hardware, software, or platforms that they can use in their project (~1 hour)</td>
+    </tr>
+    <tr>
+      <th>DIY Careeer Info Session</th>
+      <td>Your company can host a career info session to introduce your company to the hackers and give out professional development tips</td>
+    </tr>
+    <tr>
+      <th>Company Challenges</th>
+      <td>Your company will be able to design challenges for hackers to tackle and set up prizes for distinguished projects.</td>
+    </tr>
+  </table>
+</div>
+
+
+    <!-- SOCIALS -->
```

Comment:
Instead of these, let's put in this info:
<img width="882" alt="image" src="https://github.com/HKN-UCSD/hard-hack-2024/assets/103216376/83a4b6b8-1b3b-4c47-9fd9-1ac13be9078e">